services/proxy/README.md

```diff
@@ -0,0 +1,19 @@
+# Proxy Service
+
+The proxy service is an API-Gateway for the ownCloud Infinite Scale microservices. Every HTTP request goes through this service.
+Authentication, logging and other preprocessing of the requests also happen here. The proxy doesn't implement things like request rate limitting
+or intrusion prevention. Those mechanisms can be achieved by setting up systems in front of the proxy service like a reverse proxy.
```

Comment:
```suggestion
The proxy service is an API-Gateway for ownCloud Infinite Scale microservices. Every HTTP request goes through this service. Authentication, logging and other preprocessing of requests also happens here. Mechanisms like request rate limitting or intrusion prevention are **not** included in the proxy service and must be setup in front like with an external reverse proxy.

The proxy service is the only service communicating to the outside and needs therefore usual protections against DDOS, Slow Loris or other attack vectors. All other services are not exposed to the outside, but also need protective measures when it comes to distributed setups like when using container orchestration.
```
* Every HTTP request...
means internal and external? If yes we should add this.

Reply:
If there is also internal http, tls is an important method to secure communication when using eg kubernetes. If there is no internal http traffic, the focus is on securing external traffic only.

Reply:
Well, there is internal HTTP traffic which will be protected via TLS but that is independent of the requests going through the proxy or not.